projects/bootloader/src/query.c

```diff
@@ -39,6 +39,16 @@ static CanDatagramTxConfig s_response_config = {
   .tx_cmpl_cb = tx_cmpl_cb,
 };
 
+static size_t prv_strnlen(const char *str, size_t maxlen) {
```

Comment:
This looks nice and clean, if you find yourself using this again in another file it might be worth moving it to the libcore library.